### DISCUSSION TITLE
Do we have a way to add InputLeftElement ?

### DISCUSSION THREAD

**Discussion by zu1k** [csandman/chakra-react-select#291] [Q&A] [answered]
When using Chakra `Input`, we have `InputGroup` and we can add an `Icon` as `InputLeftElement` to its left.

Do we have a way to add `InputLeftElement` or `SelectorLeftElement` to our `Selector`?

**Comment by egerpe-ats** [comment]
Hi @zu1k, I'm trying the same thing. Did you find a solution?

**Reply by zu1k** [reply, marked as answer]
See https://github.com/zu1k/bs-core/blob/098296440e76b868cf04f3f543e7fa4bbbeff6f2/frontend/src/components/SearchInput.tsx#L65

**Reply by csandman** [reply]
Thanks for coming back! I must have missed this when you originally posted it, sorry about that.
